maxmin/max3/max3.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func max(a, b float64) float64 {
```

Comment:
nit: Please try to avoid using features that we haven't discussed yet. Our tasks are designed to ensure they could be solved using only topics discussed.